plugin-flex-ts-template-v2/src/feature-library/contact-history/flex-hooks/strings/index.ts

```diff
@@ -0,0 +1,61 @@
+export enum StringTemplates {
+  ContactHistory = 'PSContactHistory',
+  ClearHistory = 'PSClearHistory',
+  ClearHistoryDialog = 'PSClearHistoryDialog',
+  ContactChannel = 'PSContactChannel',
+  ContactInboundCall = 'PSContactInboundCall',
+  ContactOutboundCall = 'PSContactOutboundCall',
+  ContactTwilioPhoneNumber = 'PSTwilioPhoneNumber',
+  ContactCustomerPhoneNumber = 'PSContactPhoneNumber',
+  OutboundCallDialog = 'PSOutboundCallDialog',
+  OutboundCallerId = 'PSOutboundCallerId',
+  ContactName = 'PSContactName',
+  ContactDateTime = 'PSContactDateTime',
+  ContactDuration = 'PSContactDuration',
+  ContactQueue = 'PSContactQueue',
+  ContactCallRecording = 'PSContactCallRecording',
+  ContactOutcome = 'PSContactOutcome',
+  ContactNotes = 'PSContactNotes',
+}
+
+export const stringHook = () => ({
+  'en-US': {
+    [StringTemplates.ContactHistory]: 'Contact History',
+    [StringTemplates.ClearHistory]: 'Clear History',
+    [StringTemplates.ClearHistoryDialog]: 'Please confirm that you want to delete all your contact history.',
+    [StringTemplates.ContactChannel]: 'Channel',
+    [StringTemplates.ContactInboundCall]: 'Inbound Call',
+    [StringTemplates.ContactOutboundCall]: 'Outbound Call',
+    [StringTemplates.ContactCustomerPhoneNumber]: 'Customer Phone Number',
+    [StringTemplates.ContactTwilioPhoneNumber]: 'Twilio Number',
+    [StringTemplates.OutboundCallDialog]: 'Please confirm you want to call',
+    [StringTemplates.OutboundCallerId]: 'Caller Id',
+    [StringTemplates.ContactName]: 'Name',
+    [StringTemplates.ContactDateTime]: 'Date & Time',
+    [StringTemplates.ContactDuration]: 'Task Duration',
+    [StringTemplates.ContactQueue]: 'Queue',
+    [StringTemplates.ContactCallRecording]: 'Call Recording',
+    [StringTemplates.ContactOutcome]: 'Outcome',
+    [StringTemplates.ContactNotes]: 'Notes',
+  },
+  'es-MX': {
+    [StringTemplates.ContactHistory]: 'Historial de contactos',
+    [StringTemplates.ClearHistory]: 'Borrar la historia',
+    [StringTemplates.ClearHistoryDialog]: 'Confirme que desea eliminar todo su historial de contactos.',
+    [StringTemplates.ContactChannel]: 'Canal',
+    [StringTemplates.ContactInboundCall]: 'Llamada entrante',
+    [StringTemplates.ContactOutboundCall]: 'Llamada saliente',
+    [StringTemplates.ContactCustomerPhoneNumber]: 'Número de teléfono del cliente',
+    [StringTemplates.ContactTwilioPhoneNumber]: 'Número de Twilio',
+    [StringTemplates.OutboundCallDialog]: 'Por favor confirma que quieres llamar',
+    [StringTemplates.OutboundCallerId]: 'Identificador de llamadas',
+    [StringTemplates.ContactName]: 'Nombre',
+    [StringTemplates.ContactDateTime]: 'Fecha y Hora',
+    [StringTemplates.ContactDuration]: 'Duración de la tarea',
+    [StringTemplates.ContactQueue]: 'Cola',
+    [StringTemplates.ContactCallRecording]: 'Grabacion de llamada',
+    [StringTemplates.ContactOutcome]: 'Resultado',
+    [StringTemplates.ContactNotes]: 'Notas',
+  },
```

Comment:
In other features, translations are in an external file. We should be consistent.